src/web/command_control/client/components/TaskSettingsPanel.tsx

```diff
@@ -81,6 +83,91 @@ function TaskOptionsPanel(props: Props) {
         props.onChange(newTask)
     }
 
+    /**
+     * Handles the change event for bottom dive parameters.
+     * 
+     * @param {React.ChangeEvent<HTMLInputElement>} evt The event object containing information about the input change.
+     * @returns {void}
+     */
+    function onChangeBottomDiveParameter(evt: React.ChangeEvent<HTMLInputElement>) {
+        const target = evt.target as any;
+        const key = target.name as (keyof DriftParameters)
+        const value = Number(target.value);
+
+        var newTask = deepcopy(task);
+        newTask.surface_drift[key] = value;
+        GlobalSettings.driftParameters[key] = value
+        Save(GlobalSettings.driftParameters)
+
+        // Array of dive parameters and fixed values
+        const keys: (keyof DiveParameters)[] = ['depth_interval', 'hold_time', 'max_depth'];
+        const values = [60, 60, 0];
+    
+        // Assigning fixed values to diveParameters
+        for (let i = 0; i < keys.length; i++) {
+            const k = keys[i];
+            const v = values[i];
+            GlobalSettings.diveParameters[k] = v;
+        }
+        Save(GlobalSettings.diveParameters)
+
+        props.onChange(newTask);
+    }
+
+    /**
+     * Checks if the Bottom dive is currently checked.
+     * 
+     * @returns {boolean} True if Bottom dive is checked, false otherwise.
+     */
+    function isBottomDiveChecked(){
+
+        if (task?.bottom_dive === undefined) {
+            return false
+        }
+        return task.bottom_dive
+    }
+
+    /**
+     * Handles the toggle action for the dive task.
+     * Also calls sets default values for diveparameters.
+     * 
+     * @returns {void}
+     */
+    function handleToggle(){
+
+        if (task?.bottom_dive === undefined) {
+            task["bottom_dive"] = true
+        } else if (task.bottom_dive) {
+            task.bottom_dive = false
+        } else {
+            task.bottom_dive = true
+        }
+
+        if(task.bottom_dive){
+
+        var newTask = deepcopy(task);
+        newTask.surface_drift['drift_time'] = surface_drift.drift_time;
+        GlobalSettings.driftParameters['drift_time'] = surface_drift.drift_time
+        Save(GlobalSettings.driftParameters)
+        props.onChange(newTask);
+
+        // Array of dive parameters and fixed values
+        const keys: (keyof DiveParameters)[] = ['depth_interval', 'hold_time', 'max_depth'];
+        const values = [60, 60, 0];
+    
+        // Assigning fixed values to diveParameters
+        for (let i = 0; i < keys.length; i++) {
```

Comment:
I see what you are doing here but it may be a little clearer if we just set depth_interval, hold_time, max_depth on their own lines. The number code lines should not change after removing the for loop.